src/identity_committer/tasks/process_identities.rs

```diff
@@ -74,6 +74,8 @@ impl IdentityCommitter {
                         &updates
                     ).await?;
 
+                    Self::log_batch_size(updates.len()).await?;;
```

Comment:
We could move this inside of `commit_identitites` and then we wouldn't need to have this kind of duplication.